### DISCUSSION TITLE
/hyndsight/beamer-handout/

### DISCUSSION THREAD

**Discussion by robjhyndman** [robjhyndman/robjhyndman.com#190] [Announcements]
Creating a handout from beamer slides 
 https://robjhyndman.com/hyndsight/beamer-handout/

**Comment by robjhyndman** [comment]
```header
Originally posted by Ken on 2014-06-11 08:37:31
````
To get multiple slides in a pdf you don\'t need a real computer. Just
install pdfcreator and then open the pdf using Acrobat Reader, print,
select multiple and print using pdfcreator. On a Mac you can probably
need pdfnup, as Reader doesn\'t print to pdf

**Comment by neel** [comment]
Unfortunately pdfnup is broken in my machine. I get `ModuleNotFoundError: No module named 'PyPDF2.pdf'` I have PyPDF2 installed. But I don't know if something module named `PyPDF2.pdf` exists.

**Reply by robjhyndman** [reply]
You may be using pdfnup from https://pypi.org/project/pdfnup/. The one I referred to is now hosted here: https://github.com/rrthomas/pdfjam-extras/blob/master/bin/pdfnup
